numpy/core/src/private/ufunc_override.c

```diff
@@ -22,8 +22,8 @@
  * nor to the default __array_ufunc__ method, so instead we import locally.
  * TODO: Can this really not be done more smartly?
  */
-static PyObject *
-get_non_default_array_ufunc(PyObject *obj)
+NPY_NO_EXPORT PyObject *
+PyUFuncOverride_GetNonDefaultArrayUfunc(PyObject *obj)
 {
     static PyObject *ndarray = NULL;
```

Comment:
out of scope for this PR: Can't you just use `PyArray_Type` here?

Reply:
To use `PyArray_Type` needs the umath/multiarray merge...

Reply:
This is resolved? Github really doesn't help here, can the committer hit the resolve button? Should they?